File: SharingMyWishList/app/src/main/java/com/example/sharingmywishlist/Response/WishAllResponse.java
package com.example.sharingmywishlist.Response;

import java.util.List;

public class WishAllResponse {

    // id
    int id;

    // title
    String title;

    // contents
    String contents;

    // writer
    String writer;

    // clear
    boolean clear;

    // color
    String color;

    public int getId() {
        return id;
    }

    public String getTitle() {
        return title;
    }

    public String getContents() {
        return contents;
    }

    public String getWriter() {
        return writer;
    }

    public boolean isClear() {
        return clear;
    }

    public String getColor() {
        return color;
    }
}

File: SharingMyWishList/app/src/main/java/com/example/sharingmywishlist/Response/WIshAllResponse.kt
package com.example.sharingmywishlist.Response

import com.google.gson.annotations.SerializedName

data class WishAllResponse (
    @SerializedName("wishResponseList") val wishResponseList: List<WishResponseList>
) {
    data class WishResponseList (
        val id: Int,
        val title: String,
        val contents: String,
        val writer: String,
        val clear: Boolean,
        val color: String
    )
}